src/coreclr/jit/llvm.cpp

```diff
@@ -1591,6 +1613,13 @@ Value* Llvm::localVar(GenTreeLclVar* lclVar)
     return llvmRef;
 }
 
+void Llvm::buildLocalVarAddr(GenTreeLclVar* lclVar)
```

Comment:
Nit: `lclVar` -> `lclAddr`.

We should handle `GT_LCL_FLD_ADDR` here too via a GEP.

Reply:
I added some code for `GT_LCL_FLD_ADDR ` but I'm not sure that it actually brings in any new methods to the compilation.